python/ray/_private/test_utils.py

```diff
@@ -418,16 +437,19 @@ def run_string_as_driver_nonblocking(driver_script, env: Dict = None):
             'exec("del script\\n" + script)',
         ]
     )
-    proc = subprocess.Popen(
-        [sys.executable, "-c", script],
-        stdin=subprocess.PIPE,
-        stdout=subprocess.PIPE,
-        stderr=subprocess.PIPE,
-        env=env,
-    )
-    proc.stdin.write(driver_script.encode("ascii"))
-    proc.stdin.close()
-    return proc
+    # Rich log handler is intended for human-readability. There are a lot of tests
+    # which inspect subprocess output for validity, so force plain logging here.
+    with temporary_environment_append({"FORCE_PLAIN_LOGGING": "1"}):
```

Comment:
Instead of this, can you just check the env var from the logger or sth? 

We export `BUILDKITE="true"` to the CI tests

Reply:
Done.